### ISSUE TITLE
Errors setting up app with redux

### ISSUE THREAD

**Issue by JulianKingman** [react-navigation/react-navigation-redux-helpers#2]
I set up a project with redux, and separated things into different files for organizational purposes, and was getting all kinds of errors, mainly `Navigator.router.getStateForAction` returning null/undefined. So I set up a snack to reproduce the issue, but to start with, followed the documentation example exactly, but I can't get it to run (it says `failed to import module 'ren'`, etc...).

Here's the snack repo: https://snack.expo.io/@buishi/react-navigation-redux-bug

And the original issue on react navigation: https://github.com/react-navigation/react-navigation/issues/3421

**Comment by Ashoat** [comment]
The `failed to import module 'ren'` issue seems to be related to expo. It happens even when I try and run a simple project.

Could you maybe make a basic RN example and put up it up on GitHub? You can run `react-native init` to create basic RN project. Guide is under "Building Projects with Native Code" [here](https://facebook.github.io/react-native/docs/getting-started.html).

**Comment by JulianKingman** [comment]
I set up a separate issue for the snack issue: https://github.com/expo/expo/issues/1418

After pulling the locally and testing it, it appears to work fine (after some hacking). It looks like my issues were probably related to load order, which I had to carefully dissect. I'm still not sure why the router was coming up blank, maybe it will become clear comparing to my other project in which the issue first appeared.
